src/components/App/Issue/VoteFinalize/Steps/Step6.js

```diff
@@ -22,28 +22,30 @@ const Step6 = ({ updateCurrentStep }) => {
         <div>
           <SubHeader>Register Public Voter</SubHeader>
           <OrderedList>
-            <li>Navigate to register voter. <LinkText onClick={() => window.location.href = 'zerotheft://register-voter?holonRedirectUrl=' + encodeURIComponent(buildUrl())}>Register Voter</LinkText></li>
+            <li>Navigate to register voter. <LinkText onClick={() => window.open('https://app.zerotheft.net/steps/')}>Register Voter</LinkText></li>
```

Comment:
Use URL in constants, think of staging, private and other environment mode where URL would be different